common/men_at_arms_types/00_maa_types.txt

```diff
@@ -287,6 +384,33 @@ bombard = {
 	
 	stack = 10
 	allowed_in_hired_troops = no
+	icon = cannon
+	
+}
+###WARCRAFT###
+massive_aerial_unit = {
+	type = massive_aerial
+	
+	damage = 70
+	toughness = 24
+	pursuit = 30
+	screen = 20
+	
+	terrain_bonus = {
+
+	}
```

Comment:
Shouldn't they have a minus in forest and jungle like light aerial do?

Reply:
I would imagine a dragon would have no problem burning the forest/jungle and a zeppelin would have no problem bombing everything

Reply:
Fair.